phc/easy/dstu3.py

```diff
@@ -71,9 +71,9 @@ def create(self, data: dict, auth_args: Auth = Auth.shared()):
 
         response = client._fhir_call(
             f"{self.entity}", http_verb="POST", json=data
-        ).data
+        )
 
-        if response == "Created":
+        if response.status_code == 201:
             return True
```

Comment:
I know it's a change to the contract, but I'd prefer getting the response.data back. It should evaluate to True if any caller is testing it (unless it returns an empty dict or something).
```suggestion
        if 200 <= response.status_code < 300:
            return response.data
```